packages/fast-usdc/src/types.ts

```diff
@@ -78,14 +78,15 @@ export interface FeedPolicy {
   eventFilter?: string;
 }
 
-export type FastUSDCConfig = Passable & {
+export type FastUSDCConfig = {
   terms: FastUsdcTerms;
   oracles: Record<string, string>;
   feeConfig: FeeConfig;
   feedPolicy: FeedPolicy & Passable;
+  noNoble: boolean; // support a3p-integration, which has no noble chain
   chainInfo: Record<string, CosmosChainInfo & Passable>;
   assetInfo: [Denom, DenomDetail & { brandKey?: string }][];
-};
+} & CopyRecord;
```

Comment:
consider putting it at the front (like `Passable &` was) so it's more noticeable.

Reply:
Why would we want it to be noticeable? It's only here due to limitations in tsc or the Passable type or something.

Reply:
> Why would we want it to be noticeable?
Because it's unexpected